helper-scripts/ansible/roles/mailcow-dockerized/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: Make sure git is installed
+  package:
+    name: git
+    state: present
+
+- name: Clone the master branch of mailcow-dockerized repo
+  git:
+    repo: https://github.com/mailcow/mailcow-dockerized
```

Comment:
This is not going to work, since this is not the git url. It should be: `https://github.com/mailcow/mailcow-dockerized.git`

Reply:
For some reason it worked for me , but in any case you are right, the correct syntax is as you say, so I fixed this